agent/structs/acl.go

```diff
@@ -1838,3 +1839,51 @@ func (id *AgentRecoveryTokenIdentity) IsLocal() bool {
 func (id *AgentRecoveryTokenIdentity) EnterpriseMetadata() *acl.EnterpriseMeta {
 	return nil
 }
+
+const ServerManagementToken = "server-management-token"
```

Comment:
This is the Accessor ID, not the Secret ID right? For clarity, I wonder if we should call this something like `ServerManagementTokenAccessorID`?